constellations/gemini-admin/src/emails/email.js

```diff
@@ -0,0 +1,241 @@
+"use strict";
+require("./email.sass");
+import React from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import API, { executeApiCalls } from "../api.js";
+import { getCurrentAccountId, setCurrentAccountId } from "../localStorage.js";
+import { Modal } from "../modals/modal.js";
+import { YesNoModal } from "../modals/yesnoModal.js";
+import { AccountInfo } from "./accountInfo.js";
+
+export class PaymentEmails extends React.Component {
```

Comment:
name this file emailPayments.js. There are going to be other email pages, so we want to differentiate them